src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -234,6 +239,20 @@ public void init(AdaptorContext context) throws Exception {
       }
     }
 
+    if (!isNullOrEmptyString(cfg.getValue("db.extraMetadataSql"))) {
+      extraMetadataSql = cfg.getValue("db.extraMetadataSql");
+      log.config("extra metadata sql: " + extraMetadataSql);
+      String extraMetadataColumnsConfig =
+          cfg.getValue("db.extraMetadataColumns");
+      if ("".equals(extraMetadataColumnsConfig)) {
+        extraMetadataColumns = null;
+        log.config("extra metadata columns: Use all columns in ResultSet");
+      } else {
+        extraMetadataColumns = new MetadataColumns(extraMetadataColumnsConfig);
+        log.config("extra metadata columns: " + extraMetadataColumns);
+      }
+    }
```

Comment:
Add an else to check for non-empty db.extraMetadataColumns and db.extraMetadataSqlParameters and add them to ignored. You should add a test for that as well (I think ignoring both in one test is fine, or maybe even add it to an existing ignored property test; I'm not sure how those are organized.